src/pages/registration/contributors_tab/components/Contributors.tsx

```diff
@@ -44,12 +44,21 @@ export const Contributors = ({ contributorRole = ContributorRole.CREATOR }: Cont
     setFieldValue(ContributorFieldNames.CONTRIBUTORS, contributorsRef.current);
   }, [setFieldValue]);
 
+  const getContributorHeading = (contributorRole: ContributorRole) => {
```

Comment:
kunne godt flytta denne til `contributorTranslations.ts`, så slipper vi at den opprettes på nytt for hver render også

Reply:
Ja, det er sikkert lurt. Du må gjerne ta over denne PRen 👍🏼